### PR TITLE
Add support for Templated Countdown Target Date / Time

### PR DESCRIPTION
**Pull request by KMurphs**
There are use cases where a countdown target should be dynamically set. E.g, while monitoring some testing activities that are supposed to take x amount of minutes.
Proposed solution is to use a dashboard variable to accomplish this. E.g

1. Define a variable for the dashboard (i.e. the end time for the countdown):
    - Name: countdown_target
    - Type: Constant
    - Label: 
    - Description:
    - (Default) Value:

2. Use this variable in the panel setting:
    - Set 'Countdown > EndTime' as ${countdown_target}

3. Use the panel with the variable as in 
```
url = http://localhost:3000/d/hDzucFh7z/new-dashboard?orgId=1&var-countdown_target=2021-12-06T22:04:19.019Z
```

This will preserve initial functionality, i.e. when a hard-coded value is set for the countdown

### CODE REVIEW

File: src/ClockPanel.tsx [hunk 1]
```diff
@@ -109,7 +109,7 @@ class UnthemedClockPanel extends PureComponent<Props, State> {
     }
 
     const timeLeft = moment.duration(
-      moment(countdownSettings.endCountdownTime).utcOffset(this.getTZ(timezone).format('Z'), true).diff(now)
+      moment(this.props.replaceVariables(countdownSettings.endCountdownTime)).utcOffset(this.getTZ(timezone).format('Z'), true).diff(now)
```
Comment (leventebalogh) [inline src/ClockPanel.tsx:112]: **Linting:** you will need to format it in the following way to make the tests pass:

 ```suggestion
      moment(this.props.replaceVariables(countdownSettings.endCountdownTime))
        .utcOffset(this.getTZ(timezone).format('Z'), true)
        .diff(now)
```

**Review by leventebalogh — COMMENTED** [review]
Nice, thanks a lot for the contribution @KMurphs! 👏  

Tested it out and it's working as expected. I left one comment regarding the failing build, plus I think it would be good to add a few lines in the README so it's easier to discovered and used by others.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "clock-panel",
-  "version": "1.2.0",
+  "version": "1.2.1",
```
Comment (jackw) [inline package.json:3]:
```suggestion
  "version": "1.3.0",
```

**Review by jackw — APPROVED** [review]
LGTM! 🚀 

Thanks so much for the contribution @KMurphs. Have made one minor suggestion as we try to follow semver as much as possible with versioning this plugin.

**Comment by gigillo73** [comment]
Hi. I would extract a value from a MySQL database; if this value is not null, I would use it to start the countdown. The use case is referred to the load control at home. My device read the power from the meter and receive the seconds before the switch off. I write this value into a MySQL database. I need to show this countdown (tipically 2 hours or 2 minutes) on the grafana clock panel . Thanks.

**Pull request merged**